src/interceptors/filter.interceptor.ts

```diff
@@ -1,6 +1,9 @@
 import { Interceptor, InterceptorInterface, Action } from 'routing-controllers';
 // TODO: Look for another possible solution with TS
-const { SPFDefaultParams, SensitiveParamFilter } = require('@amaabca/sensitive-param-filter')
+const {
+  SPFDefaultParams,
+  SensitiveParamFilter
+} = require('@amaabca/sensitive-param-filter');
```

Comment:
Did you consider making our own implementation of this filter?

Reply:
Yes, it's not such a big logic but it would take time... I rather have the behavior now and sent some time in the future creating this library, since it looks like it's not maintained for typescript, wdyt?